ListOfQuestions.md

```diff
@@ -14,4 +14,13 @@
 				(if	(empty?	in)
 						out
 						(recur	(rest	in)
-						(conj out (str	"Alice	is	"	(first	in)))))))
\ No newline at end of file
+						(conj out (str	"Alice	is	"	(first	in)))))))
+
+Week1:
+* Reader Macro
```

Comment:
`->>` is not a reader macro, it's just a macro.
http://clojure.org/guides/threading_macros